generated/nidcpower/session.py

```diff
@@ -185,7 +185,7 @@ class _SessionBase(object):
         session['0,1'].current_level = var
         var = session['0,1'].current_level
     '''
-    current_level_autorange = attributes.AttributeEnum(attributes.AttributeViInt32, enums.CurrentLevelAutorange, 1150017)
+    current_level_autorange = attributes.AttributeViInt32(1150017)
```

Comment:
Does this work correctly? We want clients to pass a bool, and then we turn around and pass a ViInt32 to the C API.